### ISSUE TITLE
已跑通的一个requirments.txt

### ISSUE THREAD

**Issue by XuJianzhi** [X-LANCE/AniTalker#51]
absl-py==2.3.1
aiofiles==24.1.0
aiohappyeyeballs==2.6.1
aiohttp==3.13.2
aiosignal==1.4.0
annotated-doc==0.0.4
annotated-types==0.7.0
antlr4-python3-runtime==4.9.3
anyio==4.12.0
async-timeout==5.0.1
attrs==25.4.0
audioread==3.1.0
brotli==1.2.0
certifi==2025.11.12
cffi==2.0.0
charset-normalizer==3.4.4
ci-sdr==0.0.2
click==8.3.1
ConfigArgParse==1.7.1
ctc_segmentation==1.7.4
Cython==3.2.3
decorator==4.4.2
Distance==0.1.3
dlib==19.24.5
editdistance==0.8.1
einops==0.8.1
espnet==202301
espnet-tts-frontend==0.0.3
exceptiongroup==1.3.1
fast-bss-eval==0.1.3
fastapi==0.125.0
ffmpy==1.0.0
filelock==3.20.1
frozenlist==1.8.0
fsspec==2025.12.0
g2p-en==2.1.0
gradio==6.1.0
gradio_client==2.0.1
groovy==0.1.2
grpcio==1.76.0
h11==0.16.0
h5py==3.15.1
hf-xet==1.2.0
httpcore==1.0.9
httpx==0.28.1
huggingface-hub==0.36.0
humanfriendly==10.0
hydra-core==1.3.2
idna==3.11
ImageIO==2.37.2
imageio-ffmpeg==0.6.0
importlib-metadata==4.13.0
inflect==7.5.0
jaconv==0.4.1
jamo==0.4.1
Jinja2==3.1.6
joblib==1.5.3
kaldiio==2.18.1
lazy_loader==0.4
librosa==0.10.2.post1
llvmlite==0.41.1
Markdown==3.10
markdown-it-py==4.0.0
MarkupSafe==3.0.3
mdurl==0.1.2
more-itertools==10.8.0
moviepy==1.0.3
msgpack==1.1.2
multidict==6.7.0
nltk==3.9.2
numba==0.58.0
numpy==1.22.3
omegaconf==2.3.0
opt_einsum==3.4.0
orjson==3.11.5
packaging==25.0
pandas==2.0.3
pillow==12.0.0
platformdirs==4.5.1
pooch==1.8.2
proglog==0.1.12
propcache==0.4.1
protobuf==3.20.1
pycparser==2.23
pydantic==2.12.4
pydantic_core==2.41.5
pyDeprecate==0.3.2
pydub==0.25.1
Pygments==2.19.2
pypinyin==0.44.0
python-dateutil==2.9.0.post0
python-multipart==0.0.21
python-speech-features==0.6
pytorch-lightning==1.7.7
pytorch-wpe==0.0.1
pytz==2025.2
pyworld==0.3.5
PyYAML==6.0.3
regex==2025.11.3
requests==2.32.5
rich==14.2.0
safehttpx==0.1.7
scikit-learn==1.6.1
scipy==1.7.3
semantic-version==2.10.0
sentencepiece==0.2.1
shellingham==1.5.4
six==1.17.0
soundfile==0.13.1
soxr==1.0.0
starlette==0.50.0
tensorboard==2.20.0
tensorboard-data-server==0.7.2
threadpoolctl==3.6.0
tokenizers==0.12.1
tomlkit==0.13.3
torch==1.11.0+cu113
torch-complex==0.4.4
torchaudio==0.11.0+cu113
torchmetrics==0.7.0
torchvision==0.12.0+cu113
tqdm==4.66.4
transformers==4.19.2
typeguard==4.4.4
typer==0.20.0
typing-inspection==0.4.2
typing_extensions==4.15.0
tzdata==2025.3
Unidecode==1.4.0
urllib3==2.6.2
uvicorn==0.38.0
Werkzeug==3.1.4
yarl==1.22.0
zipp==3.23.0
